subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/caching/ScriptCachingIntegrationTest.kt

```diff
@@ -227,14 +223,14 @@ class ScriptCachingIntegrationTest : AbstractScriptCachingIntegrationTest() {
             import org.gradle.api.tasks.*
 
             class MyTask extends DefaultTask {
-                static final byte[] MEMORY_HOG = new byte[64 * 1024 * 1024]
+                static final byte[][] MEMORY_HOG = new byte[1024][1024 * 64]
```

Comment:
According to https://stackoverflow.com/questions/11641985/which-comes-first-in-a-2d-array-rows-or-columns

```
Array[number of arrays][how many elements in each of those arrays]
```

Will `byte[1024 * 64][1024]` be better?

Reply:
Yes, the current code is as intended.
Your suggestion would make more smaller arrays, is that the motivation of your comment?

Reply:
Yes, I'm wondering if more smaller arrays would make allocation easier. However, arrays of 64K might be good, too.